src/translations/en.json

```diff
@@ -1631,7 +1631,7 @@
         "devices": {
           "add_prompt": "No {name} have been added using this device yet. You can add one by clicking the + button above.",
           "caption": "Devices",
-          "description": "Manage connected devices",
+          "description": "A searchable list of all configured devices",
```

Comment:
It is more than just a searchable list (you can change the name, assign it to areas, etc.), so I think this text proposal is too limited. I would actually stick with the current description here.

Reply:
Done.